scripts/exp/prepare-firestore-for-exp-release.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { projectRoot, readPackageJson } from '../utils';
+import { writeFile as _writeFile, readFile as _readFile } from 'fs';
+import { promisify } from 'util';
+import path from 'path';
+
+const writeFile = promisify(_writeFile);
+const readFile = promisify(_readFile);
+const packagePath = `${projectRoot}/packages/firestore`;
+
+// Prepare @firebase/firestore, so scripts/exp/release.ts can be used to release it
+export async function prepare() {
+  // Update package.json
+  const packageJson = await readPackageJson(packagePath);
```

Comment:
I don't fully follow this. Maybe that's ok, maybe we need some comments.

This seems to create a new package.json based on  `expPackageJson`, but it seems to use the dependencies and the name from the main Firestore package. If that understanding is correct, then I think we are not using the right name for firestore/lite. We should also consider using different dependencies so that we don't force webchannel-wrapper on users that don't use a bundler for their code.

Reply:
I added some comments. PTAL

Why do you think we are not using the right name for firestore/lite? It will be accessible by `@firebase/firestore/lite` and `firebase/firestore/lite`. Is it incorrect?

How do we force  `webchannel-wrapper` on users that don't use a bundler for their code?
Are you talking about people using CDN scripts? We will build `firebase-firestore-lite.js` in `firebase`, which should not contain webchannel-wrapper.

I believe firebase/exp has the same dependencies as we currently have, what different dependencies should it have?

Reply:
"@firebase/firestore/lite" and "firebase/firestore/lite" is correct.

Re: wenchannel-wrapper. It would be nice if users can develop against "firebase/firestore/lite" without depending on WebChannelWrapper. If we continue to ask users to just install the "firebase" package then this point is moot though.

Thanks for the comment update.

Reply:
I see. We won't be able to do that unless we put `firestore/lite` into its own package. IIUC, it's only an issue for Nodejs apps, right? E.g. the ones running in serverless environments that has limited storage. I don't think it matters to Web apps since unused deps will just sit in `node_modules`, not in the compiled app code.

Reply:
Correct.